### PR TITLE
Correct the API path for getting a single notification

### PR DESCRIPTION
**Pull request by nikclayton**

### CODE REVIEW

File: content/en/methods/notifications.md [hunk 1]
```diff
@@ -394,4 +394,4 @@ Invalid or missing Authorization header.
 
 {{< page-relref ref="methods/push" caption="push API methods" >}}
 
-{{< caption-link url="https://github.com/mastodon/mastodon/blob/main/app/controllers/api/v1/notifications_controller.rb" caption="app/controllers/api/v1/notifications_controller.rb" >}}
\ No newline at end of file
+{{< caption-link url="https://github.com/mastodon/mastodon/blob/main/app/controllers/api/v1/notifications_controller.rb" caption="app/controllers/api/v1/notifications_controller.rb" >}}
```
Comment (nikclayton) [inline content/en/methods/notifications.md:397]: Not sure why this change is here -- I edited the file through the Github web interface. This change can probably be deleted.

**Comment by trwnh** [comment]
fixed in #1110 as well
